kyo-scheduler/jvm-native/src/main/scala/kyo/scheduler/package.scala

```diff
@@ -1,16 +1,13 @@
 package kyo
 
-import java.util.logging.*
 import kyo.stats.internal.StatsRegistry
 
 package object scheduler {
 
-    private[scheduler] val log = Logger.getLogger("kyo.scheduler")
-
     private[scheduler] def statsScope =
         StatsRegistry.scope("kyo", "scheduler")
 
     private[scheduler] def bug(msg: String, ex: Throwable) =
-        log.log(Level.SEVERE, s"🙈 !!Kyo Scheduler Bug!!", new Exception(msg, ex))
+        new Exception(s"🙈 !!Kyo Scheduler Bug!! " + msg, ex).printStackTrace()
```

Comment:
There is! https://github.com/scala-native/scala-native-java-logging

Guess which project motivated me to make the port of this library? Kyo itself 😄

Reply:
I've opened a PR https://github.com/getkyo/kyo/pull/868

Reply:
ha! pretty cool :) It's nice to see the ecosystem growing. I was a bit skeptical of Scala Native initially but I think that was skewed by my experience working with Graal. Many Scala projects are mostly self-contained code that are trivial to port to Native if the ecosystem is more mature. Graal is great but it can also be a nightmare to set up. I'd say the odds is that kyo-tapir will have a Scala Native version before one that works on GraalVM Native 😅